RoadMap_Movie/RoadMap_Movie/DetailModule/Presenter/DetailPresenter.swift

```diff
@@ -0,0 +1,46 @@
+// DetailPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Презентер экрана с детальным описанием фильма
+final class DetailPresenter: DetailViewPresenterProtocol {
+    // MARK: - Public properties
+
+    let networkService: MovieNetworkServiceProtocol
+    var detail: Details?
+    var router: RouterProtocol?
+    var id: Int?
+    weak var view: DetailViewProtocol?
```

Comment:
```suggestion
    private weak var view: DetailViewProtocol?
```
проверь все проперти, можно некоторые сделать приватными